src/index.js

```diff
@@ -727,8 +731,10 @@ see doc: https://www.rabbitmq.com/reliability.html#consumer-side`);
                     subscriber: qualifier,
                     destination: '',
                     request: JSON.parse(message.content).data,
+                    requestSize: message.content.length,
                     executionMs: startTime ? new Date().getTime() - startTime : null,
-                    error: err
+                    error: err,
+                    errorSize: Buffer.from(JSON.stringify(serializeError(err))).length
```

Comment:
We should get the same result doing
```suggestion
errorSize: JSON.stringify(serializeError(err)).length
```

Reply:
most of the time, but not always:

<img width="276" alt="Screenshot 2023-10-02 at 14 32 04" src="https://github.com/cubyn/node-carotte-amqp/assets/8168981/73d61ba6-3c52-4603-84d0-ded688d61281">

Reply:
yes when there's some non utf-8 chars, you're right